libs/product/src/models/configurable-product.ts

```diff
@@ -37,11 +52,29 @@ export interface DaffConfigurableProductAttribute extends DaffOrderable {
  * an "available attributes" selector.
  */
 export interface DaffConfigurableProductVariant {
+	/**
+	 * The attributes applied to the product to form this product variant.
+	 */
 	appliedAttributes: DaffProductVariantAttributesDictionary;
+	/**
+	 * The id for this variant.
```

Comment:
`this variant` vs `the variant` used. (I prefer `the variant`)

Reply:
changed